common/src/main/java/org/apache/uniffle/common/ShufflePartitionedData.java

```diff
@@ -20,31 +20,37 @@
 import java.util.Arrays;
 
 import com.google.common.annotations.VisibleForTesting;
-import org.apache.commons.lang3.tuple.Pair;
+import org.apache.commons.lang3.tuple.Triple;
```

Comment:
Could we extra a class instead of using Tripple?

Reply:
I abstract a class named ShufflePartitionedBlocksInfo